testnets/safrochain/assetlist.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "safrochain",
```

Comment:
please rename the chain_name (and directory name) to `safrochaintestnet` (because testnets must end with '-testnet')